src/mocks/responses.js

```diff
@@ -79,16 +79,16 @@ export const responses = {
   // Get Dataset details (embedded) Metadata Repository Service
   "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887987": embeddedDataset[0],
   // Get Dataset details (embedded) Metadata Repository Service
-  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { let y = { ...x }; y.accession = "GHGAD588887988"; return y })[0],
+  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { let y = { ...x }; y.accession = "GHGAD588887988"; y.ega_accession = "EGAD588887988"; return y })[0],
```

Comment:
```suggestion
  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { {...x, accession: "GHGAD588887988", ega_accession: "EGAD588887988"})[0],
```

Reply:
a bit simpler, similarly below.

Reply:
maybe better to add a function in data that returns the dataset for an accession, or prepare an object with accessions as keys

Reply:
I am not sure if you can use the keys in the responses in the return object definition? I think I tried that before. In any case, I applied your suggestion